.jenkins/install_virtualenv.sh

```diff
@@ -11,20 +11,22 @@ exitError()
 
 # check a virtualenv path has been provided
 test -n "$1" || exitError 1001 ${virtualenv_path} "must pass an argument"
+pace_dir=`dirname $0`/../
```

Comment:
I am not sure that this always works, so I've been using the other pattern wherever I add new operations like this. But I've often kept the old pattern where it exists.

Should I update this to the new pattern?

Reply:
I'm ok with leaving it as is. 
I was mostly wondering because we have several new places where you've added pieces along the lines of 
```
SCRIPT_DIR="$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
PACE_DIR=$SCRIPT_DIR/../../../../
```
And we could have used that here as well. But not really necessary

Reply:
Updated.